chrome_extension/content-script.js

```diff
@@ -86,17 +93,22 @@ function applyStylesToElements(newItem, oldItem = null) {
   }
 }
 
-function removePreviouslyExcludedStyles() {
-  document.querySelectorAll('.web-scraper-helper-exclude').forEach(e => {
-    if (e && e.classList) {
-      e.classList.remove('web-scraper-helper-exclude');
-    }
-  });
+function removePreviouslyExcludedStyles(parentSelector = null) {
```

Comment:
In this case, we want to remove the class in the whole page, we don't need to consider the `parentSelector`. You can revert this function.

Reply:
This function is called when a subItem config file is closed too. Hence I used the parent selector here.

Reply:
🤔 Does it mean the excluded items from subitems aren't hidden if we aren't in the tab for subitems?
I think we should hide them generally. We can discuss it.